config.php

```diff
@@ -60,6 +60,25 @@
 COMMENT
 			,
 		],
+		
+		/**
+		 * missing formatting
+		 */
+		[
+			'label' => 'missing formatting',
+			'action' => 'comment',
+			'comment' => <<<COMMENT
+Thank you for your issue.
+
+Unfortunately code that is not formatted as code blocks is extremely hard to read.
+Please check GitHub formatting guide and wrap code accordingly.
```

Comment:
Would be good to link to formatting guide here as I've proposed in the issue.

Reply:
@samdark Right 😏